pages/index.js

```diff
@@ -11,118 +11,185 @@ export default function Home() {
-                  No. Facilities are prioritizing workers who can begin immediately, but will also consider
-                  recently retired individuals who are in good standing with their respective licensing board.
-                </p>
-
-                <h4>How much will I get paid? Do I have to travel?</h4>
-                <p>
-                  The specific conditions of your employment, like pay and location, will be negotiated between
-                  you and the individual facilities. Before being hired, you will have an opportunity to discuss
-                  details like this with your potential future employer.
-                </p>
-
-                <h4>I’m not looking for a job, but I want to help. What should I do? </h4>
-                <p>
-                  Please sign up to be a volunteer through the Louisiana <a href="https://www.lava.dhh.louisiana.gov/">
-                  Volunteers in Action registry</a>. Medical and non-medical volunteers 18 years of age and older are welcome.
-                </p>
+                <h4>HEALTHCARE WORKERS </h4>
+
+                <div>
+                  <h5>What workers can participate? </h5>
+                  <p>
+                    Licensed or retired, furloughed, and out-of-work healthcare
+                    workers willing to work in Louisiana can submit their
+                    information to be considered for employment.
+                  </p>
+
+                  <h5>
+                    I’m an interested healthcare worker. How do I participate?
+                  </h5>
+                  <p>
+                    Please fill in{" "}
+                    <Link href="/candidate-intake?souce=FAQ">
+                      <a>this form</a>
+                    </Link>{" "}
+                    to submit your information.
+                  </p>
+
+                  <h5>
+                    I don’t live in Louisiana but want to help. Can I
+                    participate?
+                  </h5>
+                  <p>
+                    Yes. We hope to find jobs for our furloughed neighbors so we
+                    will be prioritizing filling open positions with qualified
+                    Louisiana residents. However, if we aren’t able to fill a
+                    critical hospital need with in-state providers, we will be
+                    expanding the reach beyond state lines.
+                  </p>
+
+                  <h5>Do I need to have an active license?</h5>
+                  <p>
+                    No. Facilities are prioritizing workers who can begin
+                    immediately, but will also consider recently retired
+                    individuals who are in good standing with their respective
+                    licensing board.
+                  </p>
+
+                  <h5>How much will I get paid? Do I have to travel?</h5>
+                  <p>
+                    The specific conditions of your employment, like pay and
```

Comment:
ok, I've set up my editor to prevent myself from adding any more longer-than-80-char rows :)